sdk/maps/azure-maps-service/samples/alias.py

```diff
@@ -0,0 +1,63 @@
+# These examples are ingested by the documentation system, and are
+# displayed in the SDK reference documentation. When editing these
+# example snippets, take into consideration how this might affect
+# the readability and usability of the reference documentation.
+import argparse
+
+from azure.maps.service.operations import AliasOperations
+from azure.core.exceptions import HttpResponseError
+from typing import List
+from common.common import create_maps_client
+
+
+def create(alias: AliasOperations):
+    alias_create_response = alias.create()
+    print("Created alias with:")
+    print(alias_create_response)
+    return alias_create_response.alias_id
+
+
+def assign(alias: AliasOperations, alias_id: str, creator_data_item_id: str):
+    alias_list_item = alias.assign(alias_id, creator_data_item_id)
+    print("Assigned resource with creator_data_item_id {} to alias with alias_id: {}".format(
+        creator_data_item_id, alias_id))
+    print(alias_list_item)
+
+
+def list(alias: AliasOperations):
+    aliases = alias.list()
+    print("View all previously created aliases:")
+    for alias_list_item in aliases:
+        print(alias_list_item)
+
+
+def delete(alias: AliasOperations, alias_id: str):
+    alias.delete(alias_id)
+    print("Deleted alias with aliasId: {}".format(alias_id))
+
+
+def get_details(alias: AliasOperations, alias_id: str):
+    alias_list_item = alias.get_details(alias_id)
+    print("Got details of alias:")
+    print(alias_list_item)
+
+
+def main(creator_data_item_id: str):
```

Comment:
The examples should do a better job of explaining what the user is trying to accomplish/tell a story. The APIs that they use are so simple that the example does not provide much value (e.g. `get_details` calls `get_details` and prints the result is the sample equivalent of documenting a 'name' property as 'the name'). Additionally, we should not break out the sub-parts into individual helper methods since that makes it much harder to read the code.